docs/src/remap/remap.adoc

```diff
@@ -316,7 +316,7 @@ It is an error to omit the '<code>' parameter.
 The options of the REMAP statement are separated by whitespace. The options are
 keyword-value pairs and currently are:
 
-`modalgroup=`'<modal group>'::
+``modalgroup=``_<modal group>_::
```

Comment:
Not sure why, but ascidoc-py likes it only this way:
```suggestion
`modalgroup=`__<modal group>__::
```

Reply:
Have you tested this? On my viewer my version was fine.

Reply:
Of course. Would be nice if _you_ do this - this would save me much time and would lead to a quicker acceptance of your PRs ;-)
You remember we realized that the previewer corresponds with asciidoctor's syntax? So that is only a partial help.